test/api_tests.jl

```diff
@@ -38,17 +38,9 @@ model1 = ABM(Agent1, GridSpace((3,3)))
 agent = add_agent!((1,1), model1)
 @test agent.pos == (1, 1)
 @test agent.id == 1
-pos1 = model1.space.agent_positions[coord2vertex((1,1), model1)]
-@test length(pos1) == 1
-@test pos1[1] == 1
 
 move_agent!(agent, (2,2), model1)
-
 @test agent.pos == (2,2)
-pos1 = model1.space.agent_positions[coord2vertex((1,1), model1)]
-@test length(pos1) == 0
-pos2 = model1.space.agent_positions[coord2vertex((2,2), model1)]
-@test pos2[1] == 1
```

Comment:
Shouldn't the tests be updated instead of removed?

Reply:
Yep, I killed that too early. Will post a fix soon.